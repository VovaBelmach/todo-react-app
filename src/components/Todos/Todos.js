import TodoInput from "./todo-input/TodoInput";
import TodoList from "./todo-list/TodoList";

const Todos = () => {
  return (
    <>
      <TodoInput />
      <TodoList />
    </>
  );
};

export default Todos;
