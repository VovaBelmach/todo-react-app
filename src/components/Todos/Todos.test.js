import React from "react";
import { Provider } from "react-redux";
import { render } from "@testing-library/react";
import store from "../../stores/configureStore";
import Todos from "./Todos";

describe("Todos", () => {
  it("should renders correctly", () => {
    const { container } = render(
      <Provider store={store}>
        <Todos />
      </Provider>
    );

    // Act
    expect(container).toMatchSnapshot();
  });
});
