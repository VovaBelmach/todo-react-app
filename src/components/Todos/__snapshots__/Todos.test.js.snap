// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todos should renders correctly 1`] = `
<div>
  <div
    class="card todo-input"
  >
    <form
      aria-label="form"
    >
      <label
        class=""
      >
        <input
          class="checkbox"
          disabled=""
          type="checkbox"
        />
      </label>
      <input
        class="textInput"
        name="todo-input"
        placeholder="What should be done?"
        type="text"
        value=""
      />
    </form>
    <span
      class="error-message"
    />
  </div>
  <div
    class="card todo-list"
  >
    <p>
      All todos completed! Add a todo and it will be displayed here.
    </p>
    <footer
      class="todo-footer"
    >
      <span>
        All done!
      </span>
      <ul
        class="todo-filter"
      >
        <li>
          <button
            class="button selected"
            type="button"
            value="All"
          >
            All
          </button>
        </li>
        <li>
          <button
            class="button"
            type="button"
            value="Active"
          >
            Active
          </button>
        </li>
        <li>
          <button
            class="button"
            type="button"
            value="Completed"
          >
            Completed
          </button>
        </li>
      </ul>
      <button
        class="button todo-clear-completed"
        type="button"
        value="Clear completed"
      >
        Clear completed
      </button>
    </footer>
  </div>
</div>
`;
